public/js/script.js

```diff
@@ -442,106 +475,139 @@
       .restart();
 
     svg.attr('height', (commits.length*commitDistance)+100);
+  }
 
-    showTags = false; // TODO: animate or re-show tags after simulation finishes
+  document.getElementById('color').addEventListener('change', function() {
+    // store empty string instead of the string false, which will eval to true when recalled
+    sessionStorage.setItem('color', this.checked ? this.checked : '');
+    changeColorSetting();
   });
 
-  document.getElementById('color').addEventListener('click', function() {
-
-    const colorScale = d3.scaleOrdinal()
```

Comment:
The bodies of these functions were refactored into a helper function, which makes it look like more change than it really is.